src/assistant/Assistant.js

```diff
@@ -1,11 +1,23 @@
+// const {default: AssistantLogo} = require('../assets/top-assistant-icon.svg')
+// const {default: CloseIcon} = require('../assets/close-icon.svg')
+// const {default: ScanIcon} = require('../assets/qr-scan.svg')
+
```

Comment:
Deleting is fine, no need to comment out. I guess you either forgot or left them because this patch is still a work in progress. Both are ok btw, no problem, just leaving a note.